geometry_msgs/Point32 P
float32 v
float32 w
---
bool is_obstacle
float32 free_path_length
